distro_test.go

```diff
@@ -72,6 +72,44 @@ func isTestLinuxProcessAlive(d *wsl.Distro) bool {
 	return err == nil
 }
 
+func TestDefaultDistro(t *testing.T) {
+	want := newTestDistro(t, emptyRootFs)
+	out, err := exec.Command("wsl.exe", "--set-default", want.Name).CombinedOutput() //nolint: gosec
+	require.NoError(t, err, string(out))
+
+	got, err := wsl.DefaultDistro()
+	require.NoError(t, err, "unexpected error getting default distro %q", want.Name)
+	require.Equal(t, want, got, "Unexpected mismatch in default distro")
+}
+
+func TestDistroSetAsDefault(t *testing.T) {
+	realDistro := newTestDistro(t, emptyRootFs)
+	fakeDistro := wsl.Distro{Name: "This distro sure does not exist"}
+
+	testCases := map[string]struct {
+		distro  wsl.Distro
+		wantErr bool
+	}{
+		"happy path":  {distro: realDistro},
```

Comment:
I would be more descriptive in the test name.

"happy path" -> "succesfully set existing distro as default"
"fake distro" -> "error on setting unexisting distro as default"